src/analysis/utils/runtime-utils.ts

```diff
@@ -137,7 +137,7 @@ export const getDisplayRuntimeStatus = (status: LeoRuntimeStatus): DisplayRuntim
     ['prestarting', () => 'Resuming'],
     ['prestopping', () => 'Pausing'],
     [Utils.DEFAULT, () => _.capitalize(status)]
-  );
+  ) as DisplayRuntimeStatus; // TODO: Remove this cast.
```

Comment:
This switch case includes apparently invalid values (`leoreconfiguring`) and is missing a bunch of values (which I guess are handled by the default case). To limit changes in this PR, I left it alone.